integrated_channels/cornerstone/transmitters/content_metadata.py

```diff
@@ -16,6 +16,18 @@ def __init__(self, enterprise_configuration, client=CornerstoneAPIClient):
             client=client
         )
 
+    def transmit(self, create_payload, update_payload, delete_payload, **kwargs):
+        """
+        Placholder
+        """
+        pass
```

Comment:
lets also log that this has run and we're deliberately doing nothing, i think you can use the `_log_info` here to properly pass all the variables for splunk